### ISSUE TITLE
Feature request: water temperature in Marine Weather API

### ISSUE THREAD

**Issue by DonaldDuck313** [open-meteo/open-meteo#407]
I think it would be useful to have the weather temperature in the Marine Weather API.

**Comment by kikislater** [comment]
I think you mean SST (Sea Surface Temperature) ? 
+1 great idea

**Comment by patrick-zippenfenig** [comment]
I agree SST and wind+gusts should be available directly from the marine API. I will bundle this upgrade with more wave models in #415 

As a workaround you can use `Soil Temperature (0 cm)` with `&cell_selection=sea` to get SST.

**Comment by jonsurfs** [comment]
Hi, looking through your documentation and I do not see SST in the Marine API -- I am pretty sure I'm being done and I'm missing it. What do I call?

Also, any plans to implement great lakes temperature readings from NOAA? I'm talking about this data: https://coastwatch.glerl.noaa.gov/satellite-data-products/surface-temperature-contours-maps/?lake=m&location=1

Thank you! This is an amazing service!

**Comment by patrick-zippenfenig** [comment]
SST is not directly available through the marine API. Although I am integrating more wave models right now, they still do not include SST.

There are no plans for lake SST.

Ideally, SST could be provided by a satellite dataset with global high resolution coverage. I have seen some in the past, but did not investigate further.

**Comment by jonsurfs** [comment]
Ok, thank you! Appreciate the fast response.

**Comment by patrick-zippenfenig** [comment]
sea surface temperature is now available for testing: https://ocean-tides.open-meteo-website.pages.dev/en/docs/marine-weather-api#latitude=51.103976&longitude=1.356607&hourly=sea_surface_temperature

Data basée on Copernicus marine: https://data.marine.copernicus.eu/product/GLOBAL_ANALYSISFORECAST_PHY_001_024/description

**Comment by kikislater** [comment]
Forecast for SST, I didn't know it exists ! But I tried, how to get realtime or past data only, not forecast ?

**Comment by patrick-zippenfenig** [comment]
For realtime, the modelled SST should be fine. Copernicus uses satellite satellite data to adapt it to current conditions.

Past data for the last couple of years will be added in the next days.
